packages/query-core/src/index.ts

```diff
@@ -26,7 +26,6 @@ export {
   shouldThrowError,
 } from './utils'
 export type { MutationFilters, QueryFilters, Updater, SkipToken } from './utils'
-export { isCancelledError } from './retryer'
```

Comment:
Isn't this a breaking change? Should we just deprecate for now?

Reply:
huh, I think it was exported from the query-core only so that the framework adapters can use it. But you’re right, maybe this was used by someone to ignore cancelled errors, so let’s bring it back and deprecate it:

06e2f180